src/SerializerFactory.php

```diff
@@ -69,7 +69,8 @@ private function shouldUseObjectsForMaps() {
 	 * @return Serializer
 	 */
 	public function newEntitySerializer() {
-		$fingerprintSerializer = new FingerprintSerializer( $this->shouldUseObjectsForMaps() );
+		$options = $this->shouldUseObjectsForMaps() ? FingerprintSerializer::USE_OBJECTS_FOR_MAPS : FingerprintSerializer::USE_ARRAYS_FOR_MAPS;
```

Comment:
Please do not use C style bit fields. Either constants or short strings. The bool constants in this patch are perfectly fine. All this is package private anyway, isn't it?